sippy-ng/src/releases/ReleasePayloadTable.js

```diff
@@ -179,6 +179,12 @@ function ReleasePayloadTable(props) {
         return ''
       },
     },
+    {
+      field: 'forced',
+      headerName: 'Forced',
+      flex: 1.5,
+      hide: props.briefTable,
```

Comment:
Could we show this as a warning icon instead? And what do you think about making the row a warning color?

Patch: [0001-Show-forced-as-icon.patch.txt](https://github.com/openshift/sippy/files/8996557/0001-Show-forced-as-icon.patch.txt)


![image](https://user-images.githubusercontent.com/429763/176061230-c25a60b9-0ac9-4c20-b864-3ffc38dd9771.png)

Reply:
Yeah, that is great!